content/websockets/gateways.md

```diff
@@ -20,6 +20,8 @@ $ npm i --save-dev @types/socket.io
 $ npm i --save @nestjs/websockets @nestjs/platform-socket.io
 ```
 
+> warning **Warning** `@nestjs/platform-socket.io` currently depends on socket.io v2.3 and socket.io v3.0 client and server are not backwards compatible. However, can implement a custom adapter to use socket.io v3.0. Please refer to [this issue](https://github.com/nestjs/nest/issues/5676) for further information.
```

Comment:
```suggestion
> warning **Warning** `@nestjs/platform-socket.io` currently depends on socket.io v2.3. socket.io v3.0 is not backwards compatible with any version 2 socket.io package. If you ned socket.io v3, however, you an implement a custom adapter to use instead of the built in one. Please refer to [this issue](https://github.com/nestjs/nest/issues/5676) for further information.
```

Reply:
Thanks!